documentation/src/docs/usage/addItem().mdx

```diff
@@ -29,6 +34,12 @@ const products = [
 ]
 ```
 
+### HEADS UP
+
+When this library was made, it was done with Stripe's SKU API in mind. SKU is depreceated now in favor
+of the Price API. This is why `sku` and `id` are interchangable. You can use `sku` if you want, but it's preferred
+that you use `id` instead.
```

Comment:
If we're telling them to use `id` we should probably use it in our examples too.